Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/MainActivity.java

```diff
@@ -438,7 +439,7 @@ public void onClick(View view) {
                         .setContentIntent(pendingIntent)
                         .setAutoCancel(true);
                 NotificationManagerCompat notificationManager = NotificationManagerCompat.from(MainActivity.this);
-                notificationManager.notify(1, builder.build());
+                //notificationManager.notify(1, builder.build());
```

Comment:
Should this remain commented out? Else if this is non-functional please delete all related `NotificationManager` code

Reply:
I commented this out because it was causing a compiler error but that code was from before this PR, so I'll uncomment it.